tests/Unit/Emogrifier/HtmlProcessor/CssInlinerTest.php

```diff
@@ -1763,6 +1799,32 @@ public function emogrifyIgnoresMediaQueryWithWhitespaceOnly()
         static::assertNotContains('@media screen', $result);
     }
 
+    /**
+     * @test
+     */
+    public function emogrifyKeepsMediaRuleAfterEmptyMediaQuery()
+    {
+        $this->subject->setHtml('<html><h1></h1></html>');
+        $this->subject->setCss('@media screen {} @media all { h1 { color: red; } }');
+
+        $result = $this->subject->emogrify();
+
+        static::assertContainsCss('@media all { h1 { color: red; } }', $result);
```

Comment:
Currently, we have these helper functions in the Emogrifier and CssInliner test cases (and they probably will be useful for the other HtmlProcessor tests as well). Would you be willing to move them to a trait (in `\Pelago\Emogrifier\Tests\Support`) in a separate PR?

It looks like some unit tests for the helpers might also be helpful.